src/styles/home-page.scss

```diff
@@ -5,7 +5,8 @@ $viewport-breakpoint-s: 957px;
 $open-navigation-width: 240px;
 
 .custom-home__header {
-  background-color: cloudscape.$color-background-home-header;
+  background-color: rgb(8,60,137);
```

Comment:
This is a change in the browser appearance. While you are absolutely free to design the project UI to your needs, we would like to stay with the official AWS Cloudscape logos for the public repo.